gravitee-apim-console-webui/src/management/api/documentation-v4/components/api-documentation-v4-content-editor/api-documentation-v4-content-editor.component.html

```diff
@@ -26,10 +26,11 @@ <h4>Page content</h4>
 <div class="content__editor">
   <gio-monaco-editor
     [(ngModel)]="_value"
-    [languageConfig]="{ language: 'markdown' }"
+    [languageConfig]="getLanguageConfig()"
```

Comment:
What about setting this as a variable instead of a method? The method gets called indefinitely when in the html, and the language config shouldn't change after OnInit.

Reply:
`pageType === 'MARKDOWN' ? ({ language: 'markdown' }) : undefined`